### DISCUSSION TITLE
Welcome to balena-minecraft-server Discussions!

### DISCUSSION THREAD

**Discussion by AlexProgrammerDE** [AlexProgrammerDE/balena-minecraft-server#98] [General]
Hi guys! I just enabled the discussions tab for this project. Here you can ask, help, suggest and showcase! If you got something big or bugs please mind opening an issue instead for this! Thanks! 😸

**Comment by Iberville830** [comment]
Hello Alex, I installed the mincraft server on a Raspberry PI and it is awesome. Thank you so much. 

I can't seem to find how to put myself OP in the server and can't find any information about it. I searched for the ops.json file but cannot find it. Maybe I'm doing it wrong. 

Is there a page somewhere that can tell me how?

Thank you very much
Mat

**Reply by AlexProgrammerDE** [reply]
@Iberville830 hi there! You should see in the balena dashboard a container called "balena-rcon". Try to open via balena a terminal in it. You should be directly connected to the terminal. Then you just type `op <yourmcusername>` and press enter. Then you should have received operator permissions.

**Reply by Iberville830** [reply]
@AlexProgrammerDE It's working !!! Thanks Alex, this is a very good work you did
